mobile/src/Student/screens/StudentGameIntro/IntroInfo/index.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react'
 import { StyleSheet, Text, View, Image, Dimensions } from 'react-native'
+import { scale, verticalScale } from "react-native-size-matters"
 import { fontFamilies, fonts, colors } from '../../../../utils/theme'
 const win = Dimensions.get('window');
 
-const IntroInfo = ({ image, text }) => {
+const IntroInfo = ({ screen, Monsters, phone, sticker, text }) => {
     return (
         <View style={styles.container}>
-            <Image source={image} style={styles.image} />
+            <Image source={Monsters[0]} style={screen === 1 ? styles.monster1 : styles.monster3} />
+            <View style={styles.imgContainer}>
+                {text != "The most total points wins!" && <Image source={sticker} style={styles.phaseSticker} />}
```

Comment:
You shouldn't compare the string in this way. Also, it's error-prune as changing the text in the other screen breaks this. Instead, pass in an extra boolean parameter and use that. That'd be also more readable